### PR TITLE
rename getPrice() to getFeedPrice() and price to feedPrice

### PR DESCRIPTION
**Pull request by e18r**
Note that this could be inconsistent with daiwanese practices.

Check out also the [daiwanese version](https://github.com/makerdao/dss/pull/225)

### CODE REVIEW

**Review by gbalabasquer — APPROVED** [review]
LGTM

File: src/clip.sol [hunk 1]
```diff
@@ -210,7 +210,7 @@ contract Clipper {
     // Could get this from rmul(Vat.ilks(ilk).spot, Spotter.mat()) instead, but
     // if mat has changed since the last poke, the resulting value will be
     // incorrect.
-    function getPrice() internal returns (uint256 price) {
+    function getFeedPrice() internal returns (uint256 price) {
```
Comment (kmbarry1) [inline src/clip.sol:213]: Might as well name the return var `feedPrice`.

Comment (e18r) [reply src/clip.sol]: done!

**Review by kmbarry1 — APPROVED** [review]
I prefer a meme name but not gonna block on that.

**Pull request merged**